backend/app/views/spree/admin/tax_categories/index.html.erb

```diff
@@ -34,18 +33,47 @@
     </tr>
   </thead>
   <tbody>
-    <% @tax_categories.each do |tax_category|
-         @edit_url = edit_admin_tax_category_path(tax_category)
-         @delete_url = admin_tax_category_path(tax_category)
-    %>
+    <% @tax_categories.each do |tax_category| %>
       <tr id="<%= spree_dom_id tax_category %>" data-hook="tax_row" class="<%= cycle('odd', 'even')%>">
-        <td class="align-center"><%= tax_category.name %></td>
-        <td class="align-center"><%= tax_category.tax_code %></td>
-        <td class="align-center"><%= tax_category.description %></td>
-        <td class="align-center"><%= tax_category.is_default? ? Spree.t(:say_yes) : Spree.t(:say_no) %></td>
+        <td class="align-center">
```

Comment:
I'm strongly in favour if sticking to server side logic wherever possible. 

I think the only thing here to improve is to eliminate a bit of duplication with the `fields_for` by placing `<%= fields_for tax_category do |f| %>` above the `<tr>`